### DISCUSSION TITLE
Multiple option for CuratorPicker

### DISCUSSION THREAD

**Discussion by aymanalareqi** [awcodes/filament-curator#129] [Ideas]
ability to select multiple files from media picker would be great

**Comment by awcodes** [comment]
This has been brought up before. I just not sure of the application or api for this yet. You can also achieve the same thing with a repeater field with CuratorPickers in the schema.

**Reply by ArtMin96** [reply]
It would be great if we could choose multiple images. For example, when selecting images for a product, the repeater is not a good option.

**Reply by awcodes** [reply]
Still trying to figure this one out since the media is saved to a separate table and I'm really wanting to stay away from a polymorphic relationship, so the user can maintain their own relationships.
